From c99d1e681dc460892004054a314fa7f929f43490 Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Sat, 13 Aug 2022 10:45:59 -0500
Subject: [PATCH 005/117] dt-bindings: display: sun8i-a83t-dw-hdmi: Remove
 #phy-cells

This device is not a PHY, and none of the nodes using this schema
contain a #phy-cells property. Likely this was a copy/paste error
introduced during the YAML conversion.

Fixes: f5a98bfe7b37 ("dt-bindings: display: Convert Allwinner display pipeline to schemas")
Signed-off-by: Samuel Holland <samuel@sholland.org>
---
 .../bindings/display/allwinner,sun8i-a83t-dw-hdmi.yaml         | 3 ---
 1 file changed, 3 deletions(-)

--- a/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-dw-hdmi.yaml
+++ b/Documentation/devicetree/bindings/display/allwinner,sun8i-a83t-dw-hdmi.yaml
@@ -20,9 +20,6 @@ maintainers:
   - Maxime Ripard <mripard@kernel.org>
 
 properties:
-  "#phy-cells":
-    const: 0
-
   compatible:
     oneOf:
       - const: allwinner,sun8i-a83t-dw-hdmi
